### ISSUE TITLE
Debug mode - should data be transferred onto Alyx?

### ISSUE THREAD

**Issue by GaelleChapuis** [int-brain-lab/iblrig#78]
My understanding is that currently the data is not transferred onto Alyx if the N trial is <42 (arbitrary number).

Should we have a flag or a special mouse param file, which when selected ensures the data is *not* uploaded onto Alyx? 

This may become important for debugging the rigs (for troubleshooting we yet cannot forsee).

**Comment by nbonacchi** [comment]
I don't think so, if debugging probably the trial count is < 42 so data will not be registered anyway
If debugging the registration then you want to use the _iblrig_test_mouse as subject so we know it was a test.
no need to create more tasks
just run the task one wants to debug with _iblrig_test_mouse
if you want to test registration let it run for 43 trials if not it will be ignored
alternatively for registration we can create the transfer_me.flag in a random session and manually run the transfer command

let's not solve problems we cannot foresee, and solve problems that we have as we have them ;)

**Comment by anne-urai** [comment]
Although I like 42 for literary reasons, I think it's too liberal - there's a chance mice (especially early on) won't do many trials. Can this be reverted to 10 trials for debugging?

**Comment by GaelleChapuis** [comment]
Ten trials is way too short for debugging.

Again, instead of having a threshold for saving the data based on a number
of trial, we can have a simple flag.

**Comment by anne-urai** [comment]
Isn't running as test_mouse enough? I don't think an additional flag is needed

**Comment by GaelleChapuis** [comment]
Sure, running test_mouse could be used as the flag for not uploading the
data onto Alyx /flatiron.

**Comment by nbonacchi** [comment]
_iblrig_test_mouse is already NOT uploaded to Flatiron.
The limit of 42 is completely arbitrary but it's based on having at least some trials per each contrast condition, full contrast set is 5 or 6 contrasts 42/6 = 7 (if removing 0.5 contrast becomes 8.4 trials per condition)

**Comment by nbonacchi** [comment]
why reopened?

**Comment by GaelleChapuis** [comment]
Let's leave it open until the WG has decided whether the number 42 is a good number as threshold.
